query-service/src/main/resources/configs/common/application.conf

```diff
@@ -285,6 +285,25 @@ service.config = {
           }
         }
       }
+      {
+        name = log-event-handler
+        type = pinot
+        clientConfig = zookeeper
+        clientConfig = ${?PINOT_CONNECTION_TYPE}
+        requestHandlerInfo = {
+          tenantColumnName = tenant_id
+          viewDefinition = {
+            viewName = logEventView
+            bytesFields = ["span_id", "trace_id"]
+            fieldMap = {
+              "LOG_EVENT.timestamp_nanos": "timestamp_nanos",
+              "LOG_EVENT.spanId": "span_id",
+              "LOG_EVENT.traceId": "trace_id",
+              "LOG_EVENT.attributes": "attributes"
```

Comment:
I think no extra handling is needed for attributes field which is a json string. 
Currently map is stored in two columns, which is then combined to a Map<String, String> and then converted to a Json string, thus it needs to be specially handled.  
Attribute field is already stored as a json string (with 1 level nesting).